jekyll/_cci2/jobs-steps.md

```diff
@@ -33,89 +25,110 @@ The following diagram illustrates how data flows between jobs:
 
-Steps are a collection of executable commands which are run during a job, the `checkout:` key is required to checkout your code and a key for `run:` enables addition of arbitrary, multi-line shell command scripting.  In addition to the `run:` key, keys for `save_cache:`, `restore_cache:`,  `deploy:`, `store_artifacts:`, `store_test_results:` and `add_ssh_keys` are nested under Steps.
+Steps are a collection of executable commands which are run during a job. The `checkout:` key is required to checkout your code and the `run:` key enables addition of arbitrary, multi-line shell command scripting.  In addition to the `run:` key, keys for `save_cache:`, `restore_cache:`, `store_artifacts:`, `store_test_results:` and `add_ssh_keys` are nested under Steps. For a full list of step options see the [Configuration Reference Steps Key]({{ site.baseurl }}/2.0/configuration-reference/#steps).
 
-## Sample configuration with imported orb
-{: #sample-configuration-with-imported-orb }
+## Passing parameters to jobs
+{: #passing-parameters-to-jobs }
 
-Find full details of the AWS S3 orb in the [CircleCI Orbs Registry](https://circleci.com/developer/orbs/orb/circleci/aws-s3#commands-sync).
+Using parameters you can run a single job multiple times with different outcomes. An extension of this functionality is [matrix jobs]({{site.baseurl}}/2.0/configuration-reference/#matrix-requires-version-21). Below is a basic example of passing a parameter to a job when it is run.
 
-```yaml
+```yml
 version: 2.1
-
-orbs:
-  aws-s3: circleci/aws-s3@x.y.z #imports the s3 orb in the circleci namespace
-  # x.y.z should be replaced with the orb version you wish to use
+​
 jobs:
-  deploy2s3:
+  print-a-message:
     docker:
-      - image: cimg/<language>:<version TAG>
-        auth:
-          username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+      - image: cimg/base:2022.03
+    parameters:
+      message:
+        type: string
     steps:
-      - aws-s3/sync: #invokes the sync command declared in the s3 orb
-          from: .
-          to: "s3://mybucket_uri"
-          overwrite: true
+      - run: echo << parameters.message >>
+​
+workflows:
+  my-workflow:
+    jobs:
+      - print-a-message:
+          message: Hello!
+```
+
+## Using a job from an orb
+{: #using-a-job-from-an-orb }
+
+Orbs are packages or reusable config that you can use in your project configuration. Orbs usually contain both commands that you can use in your jobs, and whole jobs that you can schedule in your workflows. Take the [Slack orb](https://circleci.com/developer/orbs/orb/circleci/slack) as an example. This orb provides a job called [`on-hold`](https://circleci.com/developer/orbs/orb/circleci/slack#usage-on_hold_notification) which you can ue in your workflows. This job pauses the workflow to require manual approval and send a slack notification. To use this job just reference it in your workflow (see line 10):
```

Comment:
I think the word "both" here is confusing. Maybe it's just me but I read it as an orb should contain "two" commands, as well as whole jobs.